images/custom-scorecard-tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the custom-scorecard-tests binary
+FROM --platform=x86_64 golang:1.17.9 as builder
+ARG TARGETARCH
+
+WORKDIR /workspace
+
+# Copy the go source
+COPY . .
+
+# Build
+RUN GOOS=linux GOARCH=$TARGETARCH BUILD_DIR=/workspace make GO_REQUIRED_MIN_VERSION:= build/custom-scorecard-tests
+
+# Final image.
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
+
+COPY --from=builder /workspace/custom-scorecard-tests /usr/local/bin/custom-scorecard-tests
+
+ENTRYPOINT ["/usr/local/bin/custom-scorecard-tests"]
```

Comment:
Viewing from "https://github.com/openshift/cert-manager-operator/pull/64/files" page, it does not have a newline character.